File: .ProgramingBasics/SimpleConditionalStatements/07.SumSeconds/src/SumSeconds.java
import java.util.Scanner;

public class SumSeconds {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num1 = Integer.parseInt(scanner.nextLine());
        int num2 = Integer.parseInt(scanner.nextLine());
        int num3 = Integer.parseInt(scanner.nextLine());

        int seconds = num1 + num2 + num3;
        int minutes = seconds / 60;
        seconds = seconds % 60;

        if (seconds < 10) {
            System.out.printf("%d:0%d\n", minutes, seconds);
        }
        else {
            System.out.printf("%d:%d", minutes, seconds);
        }
    }
}

File: ProgramingBasics/SimpleConditionalStatements/07.SumSeconds/07.SumSeconds/SumSeconds.cs
using System;

namespace _07.SumSeconds
{
    public class SumSeconds
    {
        public static void Main(string[] args)
        {
            var num1 = int.Parse(Console.ReadLine());
            var num2 = int.Parse(Console.ReadLine());
            var num3 = int.Parse(Console.ReadLine());

            var seconds = num1 + num2 + num3;
            var minutes = seconds / 60;
            seconds = seconds % 60;

            if (seconds < 10)
            {
                Console.WriteLine($"{minutes}:0{seconds}");
            }
            else
            {
                Console.WriteLine($"{minutes}:{seconds}");
            }
        }
    }
}
